.travis.yml

```diff
@@ -10,7 +10,7 @@ env:
     - GETH_URL='https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.6.6-10a45cb5.tar.gz'
     - GETH_VERSION='1.6.6'
     - SOLC_URL='https://github.com/ethereum/solidity/releases/download/v0.4.11/solc-static-linux'
```

Comment:
Change this to `https://github.com/ethereum/solidity/releases/download/v0.4.13/solc-static-linux`